File: Denuncias.java
public class Denuncias {
    private static List<String> denuncias = new ArrayList<>();

    public static void registrarDenuncia(String denuncia) {
        // Simulação de registro de denúncia
        denuncias.add(denuncia);
    }

    public static List<String> obterDenuncias() {
        // Simulação de obtenção de denúncias
        return denuncias;
    }
}

File: denuncias.js
// Simulação de sistema de denúncias
const reportedContent = [];

function reportContent(userId, videoId, reason) {
    // Registre uma denúncia de conteúdo inadequado (simulado)
    const report = { userId, videoId, reason };
    reportedContent.push(report);
}

function handleReport(videoId, moderatorId, decision) {
    // Lide com uma denúncia e tome uma decisão (simulado)
}
